src/styles/common/_forms.scss

```diff
@@ -4,6 +4,10 @@
  * Forms
  **************************************/
 
+/* Variables */
+
+$large-pill-form-min-height: $default-line-height * 4;
```

Comment:
Wasn't sure on if to put here of in `variables.scss`. I decided to put it here because it's a bit more organized and having all shared variables in one file might not scale well.